docs/api/api/api.txt

```diff
@@ -1953,6 +1953,7 @@ XmlResult: status_ok
 
 DELETE /staging/<staging_workflow_project>/workflow
   Delete a staging workflow associated to a given project.
+XmlBody: delete_staging_workflow
```

Comment:
This is wrong. This endpoint doesn't need a body. However, there's the `with_staging_projects` option which isn't documented.

Reply:
I looked at the other `DELETE` endpoints and how we describe them. We don't refer to a `XmlBody` since it's not needed. We could be clearer by writing something like `XmlBody: none` or `XmlBody: empty`, but if we go ahead with this change, we need to be consistent and do it for all `DELETE` endpoints.

Reply:
For an example of how we describe parameters, see [this](https://github.com/openSUSE/open-build-service/blob/c6b95b130c0f2032393a324cd10a147062f3292a/docs/api/api/api.txt#L1677)